app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for someone in people:
+        name = someone["name"]
+        age = someone["age"]
+        person = Person(name, age)
+        person_list.append(person)
+
+    for someone in people:
+        if "wife" in someone and someone["wife"] is not None:
```

Comment:
You can use .get instead of usage and keyword:
someone.get("wife") is not None
The same with husband